msi/WixWin.wsi

```diff
@@ -395,6 +371,23 @@ Licensed to the Apache Software Foundation (ASF) under one or more
                                   EventMessageFile="[#MiNiFiExeWithPassword]" />
                 <Condition><![CDATA[SERVICEACCOUNT<>"LocalSystem"]]></Condition>
               </Component>
+
+              <Component Id="VSRedistributableFiles" Guid="7923f16d-785a-41cb-aa74-17e0fa5f1314">
+                <File Id="MiNiFiExe_concrt140" Name="concrt140.dll" KeyPath="no" Source="redist\concrt140.dll"/>
+                <File Id="MiNiFiExe_msvcp140" Name="msvcp140.dll" KeyPath="no" Source="redist\msvcp140.dll"/>
+                <File Id="MiNiFiExe_msvcp140_1" Name="msvcp140_1.dll" KeyPath="no" Source="redist\msvcp140_1.dll"/>
+                <File Id="MiNiFiExe_msvcp140_2" Name="msvcp140_2.dll" KeyPath="no" Source="redist\msvcp140_2.dll"/>
+                <File Id="MiNiFiExe_vccorlib140" Name="vccorlib140.dll" KeyPath="no" Source="redist\vccorlib140.dll"/>
+                <File Id="MiNiFiExe_vcruntime140" Name="vcruntime140.dll" KeyPath="no" Source="redist\vcruntime140.dll"/>
+                <?if $(env.VisualStudioVersion) = 16.0 ?>
+                  <File Id="MiNiFiExe_msvcp140_atomic_wait" Name="msvcp140_atomic_wait.dll" KeyPath="no" Source="redist\msvcp140_atomic_wait.dll"/>
+                  <File Id="MiNiFiExe_msvcp140_codecvt_ids" Name="msvcp140_codecvt_ids.dll" KeyPath="no" Source="redist\msvcp140_codecvt_ids.dll"/>
+                  <?if $(env.build_platform) = x64 ?>
+                    <File Id="MiNiFiExe_vcruntime140_1" Name="vcruntime140_1.dll" KeyPath="no" Source="redist\vcruntime140_1.dll"/>
+                  <?endif ?>
```

Comment:
Why did you remove vcruntime140_1.dll from 32bit builds? I couldn't find a description of individual DLLs.

Reply:
Because my VS2019/x86 Redist folder `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Redist\MSVC\14.28.29910\x86\Microsoft.VC142.CRT` does not contain it: 
![image](https://user-images.githubusercontent.com/7033536/119458839-73e56400-bd3d-11eb-88e1-aeff08111269.png)

I have tested this by building a package and installing it in a blank Windows docker image `mcr.microsoft.com/windows/servercore:1809`; all four combinations of 2017/2019 and x86/x64 worked (meaning the MiNiFi process started up).